templates/engage/base.html

```diff
@@ -0,0 +1,109 @@
+{% extends "engage/base_engage.html" %}
+
+{% set section_title="Engage Page" %}
+{% set section_path="/engage" %}
+
+{% block title %}ESM for Ubuntu 14.04 LTS Trusty Tahr{% endblock %}
+{% block meta_description %}Extended Security Maintenance (ESM) will be available for Ubuntu 14.04 LTS Trusty Tahr for continued security and compliance of the most popular linux operating system.{% endblock %}
+{% block meta_image %}2217d1c8-Security.svg{% endblock %}
+
+{% block meta_copydoc %}{document["metadata"]["meta_copydoc]}{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip p-engage-banner--{{document['metadata']['banner_class']}}">
+  <div class="u-fixed-width navigation-logo-engage">
+    <a href="/">
+      {% if document['metadata']['banner_class'] == 'light' %}
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/04115a7e-ubuntu_black-orange_hex.svg",
+            alt="Ubuntu",
+            height="32",
+            width="143",
+            hi_def=True,
+            loading="auto"
+          ) | safe
+        }}
+      {% else %}
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/f263d9c4-logo-ubuntu-white.svg",
+            alt="Ubuntu",
+            height="32",
+            width="143",
+            hi_def=True,
+            loading="auto"
+          ) | safe
+        }}
+      {% endif %}
+      
+    </a>
+  </div>
+  <div class="row">
+    <div class="col-7">
+      <h1>{{document["title"]}}</h1>
+      <p class="u-no-padding--top p-heading--four">Learn how to maintain ongoing security compliance for your 14.04 systems.</p>
+      {% if document["metadata"]["webinar_code"] != "" %}
+      <p>
+        <a href="#register-section" class="p-button--positive">
+          Watch the webinar
```

Comment:
- We don't have CTA text. I added that text as default now, and check if there is CTA if not, output that default text.
- Regarding link, in this specific case there is no need. It's an anchor that takes you to the brighttalk webinar section that generates the link, like in our current engage pages

Reply:
So I thought of maybe a better way to do at least the download CTA button text. It seems to be always based on the type property in the discourse data. So it's always something like Download the {type}, so you will get eventually `Download the whitepaper` or `Download the ebook`. For things like events or webinar, there is no formid, so it will not display the form.

The other CTA would be "Watch the webinar" which might need translation...

Reply:
I rectify myself, we still have the translation problem...